GOVERNANCE.MD

```diff
@@ -1,6 +1,91 @@
+
+Changes to CC membership should be posted in the agenda, and may be suggested as any other agenda item.
+
+No more than one-fourth of the CC members may be affiliated with the same employer or leadership of a community/ecosystem organization. If removal or resignation of a CC member, or a change of employment by a CC member, creates a situation where more than one-fourth of the CC membership shares an employer, then the situation must be immediately remedied by the resignation or removal of one or more CC members affiliated with the over-represented employer(s).
+
+The CC members shall consist of active members of Community Projects as defined in Section 7.
+
+The CC shall meet regularly using tools that enable participation by the community (e.g. weekly on a Google Hangout On Air, or through any other appropriate means selected by the CC). The meeting shall be directed by the CC Chairperson. Minutes or an appropriate recording shall be taken and made available to the community through accessible public postings.
+
+Section 5. Responsibilities of the CC.
+
+Subject to such policies as may be set by the Board, the CC is responsible for all cultural development and outreach within the Node.js Foundation, including:
+
+- Outreach to community organizations
+- Documentation of community organizations
+- Cultural direction of Node.js Foundation
+- Overseeing Inclusivity WG
+- Project governance and process (including this policy)
+- Recommendations for building and developing community projects that align with needs of Node.js
+- Mediating cultural conflicts between Foundation and/or community projects
+- The CC will serve as Node.js Foundation’s primary community liaison body with external open source projects, consortiums and groups.
+
+Section 6. Node.js Foundation Operations.
+
+The CC will establish and maintain a process of support for Node.js community projects. The process will establish guidelines for how culture and values of the community can be supported.
+
+
+
+The CC is responsible for vetting organizations that will be supported through means such as monetary sponsorship, promotion, resource provisioning, etc. for example requirements such as the organization in question having and executing a Code of Conduct or not conflicting with Node.js Foundation’s goals and priorities.
+
+The CC and entire technical community will follow any processes as may be specified by the Board relating to the intake and license compliance review of contributions, including the Node.js Foundation IP Policy.
+
+Section 7. Elections
+
+Leadership roles in Node.js Foundation Community Committee will be peer elected representatives of the community organizations.
+
+For election of persons (CC Chairperson, Advisors, etc.) a multiple-candidate method should be used, e.g.:
+
+- ●  Condorcet: http://en.wikipedia.org/wiki/Condorcet_method or
```

Comment:
nit: let's change `●` to `-` so that markdown will format the bullet points consistently with the other points